src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -112,6 +113,8 @@ public Command parseCommand(String userInput) throws ParseException {
         case RandomCommand.COMMAND_WORD:
             return new RandomCommandParser().parse(arguments);
 
+        case OfflineHelpCommand.COMMAND_WORD:
+            return new OfflineHelpCommandParser().parse(userInput);
```

Comment:
Great idea implementing offline help feature. Simple and easy to understand implementation.